packages/sdk/python/human-protocol-sdk/human_protocol_sdk/operator/operator_utils.py

```diff
@@ -144,7 +144,7 @@ def __init__(
         self.locked_until_timestamp = locked_until_timestamp
         self.amount_withdrawn = amount_withdrawn
         self.amount_slashed = amount_slashed
-        self.reward = reward
+        self.last_deposit_timestamp = last_deposit_timestamp
```

Comment:
This is out of context, if the entity is staker then you can understand that last_deposit_timestamp is the last time the user staked, but for an operator is not so obvious, we should think about renaming or maybe delete it. Also another option is to return an operator object that has operator.stakingInfo